daily.sh

```diff
@@ -113,7 +113,11 @@ main () {
         # only try to su if we are root (or sudo)
         if [[ "$EUID" -eq 0 ]]; then
             echo "Re-running ${DAILY_SCRIPT} as ${LIBRENMS_USER} user"
-            su -l "$LIBRENMS_USER" -c "$DAILY_SCRIPT $@"
+            if [[ "$(uname)" == "Linux" ]]; then
+                su -l "$LIBRENMS_USER" -c "$DAILY_SCRIPT $@"
+            else
+                sudo -u "$LIBRENMS_USER" "$DAILY_SCRIPT"
```

Comment:
Thanks, I had initially left it off due to it causing the script to break, but later realised it was just a quoting issue. I had intended to add it back but apparently forgot to. I will fix that now.

Regarding sudo on Linux, I'm perfectly fine to do so, but I didn't want to unnecessarily impose the additional dependency on Linux users if I didn't have to. If you're happy for everything to use sudo then i'll go ahead and remove the if statement altogether.